docs/self-hosted-appcircle/configure-server/kubernetes/helm-configuration.md

```diff
@@ -0,0 +1,702 @@
+---
+title: Helm Configuration
+description: Learn how to configure the Appcircle server Helm chart
+tags: [self-hosted, helm, configuration, kubernetes]
+sidebar_position: 1
+---
+
```

Comment:
Add some keypoints

- Values yaml update or add key
- Command to run after configuration change
- List of configuration that must be done when installation (cannot be changed later)